### PR TITLE
Remove macros: CASE, __OR, and DEFAULT

### PR DESCRIPTION
**Pull request by skitaoka**

### CODE REVIEW

**Comment by skitaoka** [comment]
Jenkins

File: toonz/sources/common/trop/tresample.cpp [hunk 1]
```diff
@@ -3225,37 +3007,6 @@ TAffine aff_0, inv_0;
 	/////////////////////////////////////////////////////////
 	// INIZIO GESTIONE ALTRI TIPI RASTER DA IMPLEMENTARE
 	/////////////////////////////////////////////////////////
```
Comment (kogaki) [inline toonz/sources/common/trop/tresample.cpp]: Accompanying comments might be removed together ?

**Comment by skitaoka** [comment]
Jenkins, again.

**Comment by kogaki** [comment]
Jenkins

**Comment by kogaki** [comment]
Jenkins

**Comment by kogaki** [comment]
LGTM

**Pull request merged**